mmv1/templates/terraform/examples/forwarding_rule_internallb_ipv6.tf.erb

```diff
@@ -0,0 +1,43 @@
+// Forwarding rule for Internal Load Balancing
+resource "google_compute_forwarding_rule" "<%= ctx[:primary_resource_id] %>" {
+  name   = "<%= ctx[:vars]['forwarding_rule_name'] %>"
+  region = "us-central1"
+
+  load_balancing_scheme = "INTERNAL"
+  backend_service       = google_compute_region_backend_service.backend.id
+  all_ports             = true
+  network               = google_compute_network.default.name
+  subnetwork            = google_compute_subnetwork.default.name
+  ip_version            = "IPV6"
```

Comment:
Agh, I just thought of a possible failure case. Some APIs, GCE in particular, will sometimes return a null when the default value is returned from the API. Could you amend an existing test to explicitly send `IPV4`? That'll confirm whether we need to mitigate that behaviour here or not.

Reply:
Got it, I'll try modify one basic test case. Bth, should we do `default_value = IPV4`, instead of `default_from_api=true`?

Reply:
Either is fine. Since the field is immutable, `default_from_api` is a little safer. Explicit default values create a slightly cleaner plan, though, since the new value is known.